src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/CustomGitSCMSource.java

```diff
@@ -33,6 +38,45 @@ public void accessibleRetrieve(@CheckForNull SCMSourceCriteria criteria, SCMHead
         super.retrieve(criteria, observer, event, listener);
     }
 
+    public SCMRevision accessibleRetrieve(SCMHead head, TaskListener listener) throws IOException, InterruptedException {
```

Comment:
See [BitbucketSCMSource](https://github.com/jaguilar-atl/atlassian-bitbucket-server-integration-plugin/pull/1/files#diff-ed9f29c11fc5b996df33a07d0643914c98ac75e7a28a3f90335963929725c5eeR321-R335) for why this is needed.